main.py

```diff
@@ -1,10 +1,52 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json") as file:
+        data = json.load(file)
+
+    for player_name, player_info in data.items():
+        race = player_info.get("race")
+        skills = player_info.get("race")["skills"]
+        guild = player_info.get("guild")
+
+        if Race.objects.filter(name=race["name"]).exists():
+            race = Race.objects.get(name=race["name"])
+        else:
+            race = Race.objects.create(name=race["name"],
+                                       description=race["description"])
+
+        for skill in skills:
+            if not Skill.objects.filter(name=skill["name"]).exists():
```

Comment:
You could get a key error, or value error, depending on which field is not provided in `skills = player_info.get("race")["skills"]` 
I mean, if you already using get() for checking if we have race field in code you could refactor a little bit this code, providing same check for skills field...